recipes/quast/meta.yaml

```diff
@@ -7,7 +7,8 @@ package:
   version: {{ version }}
 
 build:
-  number: 2
+  noarch: python
```

Comment:
This uses `pip install` with compilers, it can't be `noarch: python`

Reply:
Thanks.